tests/manifest_task_group_tests.json

```diff
@@ -3,20 +3,29 @@
     "model.dbt_test.model1": {
       "depends_on": {
         "nodes": []
+      },
+      "config": {
+        "materialized": "view"
```

Comment:
I don't see any test for ephemeral?

Reply:
Yes, I have totally forgot about writing a test for this due to scaling down from original idea. Test has unveiled uncaught error I had to fix. PR got updated.